inventory/packages/zypper.go

```diff
@@ -96,3 +97,64 @@ func ZypperUpdates() ([]PkgInfo, error) {
 	}
 	return parseZypperUpdates(out), nil
 }
+
+func parseZypperPatches(data []byte) ([]ZypperPatch, []ZypperPatch) {
+	/*
+		Repository                          | Name                                        | Category    | Severity  | Interactive | Status     | Summary
+		------------------------------------+---------------------------------------------+-------------+-----------+-------------+------------+------------------------------------------------------------
+		SLE-Module-Basesystem15-SP1-Updates | SUSE-SLE-Module-Basesystem-15-SP1-2019-1206 | security    | low       | ---         | applied    | Security update for bzip2
+		SLE-Module-Basesystem15-SP1-Updates | SUSE-SLE-Module-Basesystem-15-SP1-2019-1221 | security    | moderate  | ---         | applied    | Security update for libxslt
+		SLE-Module-Basesystem15-SP1-Updates | SUSE-SLE-Module-Basesystem-15-SP1-2019-1229 | recommended | moderate  | ---         | not needed | Recommended update for sensors
+		SLE-Module-Basesystem15-SP1-Updates | SUSE-SLE-Module-Basesystem-15-SP1-2019-1258 | recommended | moderate  | ---         | needed     | Recommended update for postfix
+	*/
+
+	lines := bytes.Split(bytes.TrimSpace(data), []byte("\n"))
+
+	var ins []ZypperPatch
+	var avail []ZypperPatch
+	for _, ln := range lines {
+		patch := bytes.Split(ln, []byte("|"))
+		if len(patch) != 7 {
+			continue
+		}
+
+		name := string(bytes.TrimSpace(patch[1]))
+		cat := string(bytes.TrimSpace(patch[2]))
+		sev := string(bytes.TrimSpace(patch[3]))
+		status := string(bytes.TrimSpace(patch[5]))
+		sum := string(bytes.TrimSpace(patch[6]))
+		switch status {
+		case "needed":
+			avail = append(avail, ZypperPatch{Name: name, Category: cat, Severity: sev, Summary: sum})
```

Comment:
Why is "not needed" not an available patch?

Reply:
Not sure it would make sense to display these, they will not be applied, they are available in the sense that they exist but are not a candidate for install. We could them if we added the status field, but I think that would just be confusing.

Reply:
SGTM. We can always extend and/or categorize these in the future.